### DISCUSSION TITLE
Where's a good place to dive in to the zod source to understand how this library works?

### DISCUSSION THREAD

**Discussion by katangafor** [colinhacks/zod#3639] [Q&A]
I've never done a deep dive into a library's source before, and it's pretty confusing for me!

Is there a good place to start to sorta understand the "core idea" of zod? Right now I'm sort of wallowing around in different files and having a hard time grasping anything.

**Comment by JacobWeisenburger** [comment]
https://youtu.be/L6BE-U3oy80?si=8PWqjfciYTHO_61f
